### PR TITLE
Public demo 07/2022

### PR DESCRIPTION
**Pull request by kpodemski**
Summary of the 7th Public Demo

### CODE REVIEW

File: news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+The project's public demo was held for the seventh time this year and [streamed live](https://www.youtube.com/watch?v=SBz7lApJ_pA) on October 5th, 2022.
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=SBz7lApJ_pA) Beginning of the session: the introduction
+- [02:11](https://youtu.be/SBz7lApJ_pA?t=131) Project news: PrestaShop 8.0 - status update
+- [03:12](https://youtu.be/SBz7lApJ_pA?t=192) Project news: Hacktoberfest 2022
+- [05:10](https://youtu.be/SBz7lApJ_pA?t=310) Project news: Q&A - new format
+- [06:04](https://youtu.be/SBz7lApJ_pA?t=364) Project news: New members in the project
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md]:
```suggestion
- [06:04](https://youtu.be/SBz7lApJ_pA?t=364) Project news: new members in the project
```

File: news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- [06:55](https://youtu.be/SBz7lApJ_pA?t=415) Pull Requests Overview
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md]:
```suggestion
- [06:55](https://youtu.be/SBz7lApJ_pA?t=415) Pull Requests overview
```

File: news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- [17:43](https://youtu.be/SBz7lApJ_pA?t=1063) Demo: Upgrade from 1.7.8 to your own PrestaShop 8.0 distribution by [@kpodemski](https://github.com/kpodemski)
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md]:
```suggestion
- [17:43](https://youtu.be/SBz7lApJ_pA?t=1063) Demo: upgrade from 1.7.8 to your own PrestaShop 8.0 distribution by [@kpodemski](https://github.com/kpodemski)
```

File: news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- [34:08](https://youtu.be/SBz7lApJ_pA?t=2048) Questions & Answers (do you have any questions? [fill this form](https://forms.gle/FWazuZnXBtFPauFZ7) and we'll answer in the next session)
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md]:
```suggestion
- [34:08](https://youtu.be/SBz7lApJ_pA?t=2048) Questions & Answers (do you have any questions? [Fill this form](https://forms.gle/FWazuZnXBtFPauFZ7) and we'll answer in the next session)
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi Krystian - just some minor suggestions.

**Comment by kpodemski** [comment]
@coffrep thank you for the super quick review! :) suggestions applied

File: news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=SBz7lApJ_pA) Beginning of the session: the introduction
+- [02:11](https://youtu.be/SBz7lApJ_pA?t=131) Project news: PrestaShop 8.0 - status update
+- [03:12](https://youtu.be/SBz7lApJ_pA?t=192) Project news: Hacktoberfest 2022
+- [05:10](https://youtu.be/SBz7lApJ_pA?t=310) Project news: Q&A - new format
+- [06:04](https://youtu.be/SBz7lApJ_pA?t=364) Project news: new members in the project
+- [06:55](https://youtu.be/SBz7lApJ_pA?t=415) Pull Requests overview
+- [17:43](https://youtu.be/SBz7lApJ_pA?t=1063) Demo: upgrade from 1.7.8 to your own PrestaShop 8.0 distribution by [@kpodemski](https://github.com/kpodemski)
+- [34:08](https://youtu.be/SBz7lApJ_pA?t=2048) Questions & Answers (do you have any questions? [Fill this form](https://forms.gle/FWazuZnXBtFPauFZ7) and we'll answer in the next session)
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-10-06-public-demo-7-2022.md]:
```suggestion
- [34:08](https://youtu.be/SBz7lApJ_pA?t=2048) Questions & Answers (do you have any questions? [Fill this form](https://forms.gle/FWazuZnXBtFPauFZ7) and we'll answer in the next session).
```

**Review by coffrep — APPROVED** [review]
Just one last change, besides that I'm approving

**Pull request merged**